endToEndTests/end_to_end_api_test.go

```diff
@@ -414,13 +415,22 @@ func TestSuccessfulEndToEndProcess(t *testing.T) {
 							}
 							So(xlsSize, ShouldBeBetweenOrEqual, 19000, 20000)
 							So(instanceResource.Downloads.XLS.Private, ShouldNotBeEmpty)
+
 							csvSize, err = strconv.Atoi(instanceResource.Downloads.CSV.Size)
 							if err != nil {
 								log.ErrorC("cannot convert csv size of type string to integer", err, log.Data{"csv_size": instanceResource.Downloads.CSV.Size})
 								t.FailNow()
 							}
 							So(csvSize, ShouldBeBetweenOrEqual, 137000, 139000)
 							So(instanceResource.Downloads.CSV.URL, ShouldNotBeEmpty)
+
+							csvwSize, err = strconv.Atoi(instanceResource.Downloads.CSVW.Size)
+							if err != nil {
+								log.ErrorC("cannot convert csv size of type string to integer", err, log.Data{"csvw_size": instanceResource.Downloads.CSVW.Size})
```

Comment:
typo (more likely copy pastor ;) ) - error message contains `csv` instead of `csvw`